### PR TITLE
Renamed nonce typo and added nonce verification.

### PR DESCRIPTION
**Pull request by Oceas**
Ticket: https://webdevstudios.atlassian.net/browse/CC-260

Description: Resolves typo in nonce_url and adds nonce verification where it previously didn't exist.

### CODE REVIEW

File: includes/class-lists.php [hunk 1]
```diff
@@ -825,7 +825,7 @@ public function check_for_list_sync_request() {
 
 		$ctct_resyncing = filter_input( INPUT_GET, 'ctct_resyncing', FILTER_SANITIZE_STRING );
 
-		if ( ! isset( $ctct_resyncing ) || ! is_admin() ) {
+		if ( ! isset( $ctct_resyncing ) || wp_verify_nonce( $ctct_resyncing, 'ctct_resyncing' ) || ! is_admin() ) {
```
Comment (richaber) [inline includes/class-lists.php:828]: @Oceas There's an error in this logic.

**Review by richaber — CHANGES_REQUESTED** [review]
It should not return when the nonce _is_ valid.

Screen Shot 2021-08-17 at 1 21 32 PM

File: includes/class-lists.php (hunk 1 above)
Comment (Oceas) [reply includes/class-lists.php]: Please find changes made. I believe laps may have been in last minute cleanups. Fresh testing and syncing works.

**Review by richaber — APPROVED** [review]
Looks good. Tested with xdebug and got the result I expected.

**Pull request merged**